### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #65: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_examples_app.py. (closes #65)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: examples/app-setup.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# quit on unbound symbols:
+set -o nounset
+
+DIR=`dirname "$0"`
+
+cd $DIR
+export FLASK_APP=app.py
+
+# Install specific dependencies
+# pip install -r requirements.txt
```
Comment (jirikuncar) [inline examples/app-setup.sh]: if we don't have any requirements then we should remove these lines

File: examples/app-setup.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+flask npm
+
+# Now install npm requirements (requires that npm is already installed):
+cd static ; npm install ; cd ..
+
+# Next, we copy the static files from the Python packages into the Flask
+# application's static folder:
+flask collect -v
+
+#Next, we build the webassets bundles:
```
Comment (jirikuncar) [inline examples/app-setup.sh]: `# Next` or `# Finally, we can build ...` (space after `#`)

File: examples/app-setup.sh (hunk 1 above)
Comment (hachreak) [inline examples/app-setup.sh]: done

File: examples/app-setup.sh (hunk 2 above)
Comment (hachreak) [inline examples/app-setup.sh]: done

**Comment by hachreak** [comment]
@jirikuncar :shipit: ?

**Pull request merged**